en/developer_guide/form_type.rst

```diff
@@ -0,0 +1,311 @@
++-----------------------------------+-----------------------------------+-----------------------+
+| ``FieldTypeType``                 | ``oo_field_type``                 |Add fields in          |
+|                                   |                                   |Content Type           |
++-----------------------------------+-----------------------------------+-----------------------+
+| ``GroupType``                     | ``oo_group``                      |Group Form             |
++-----------------------------------+-----------------------------------+-----------------------+
+| ``KeywordType``                   | ``oo_keyword``                    |Keyword Form           |
++-----------------------------------+-----------------------------------+-----------------------+
+| ``NodeType``                      | ``oo_node``                       |node Form              |
++-----------------------------------+-----------------------------------+-----------------------+
+| ``OrchestraChoiceType``           | ``choice``                        |                       |
++-----------------------------------+-----------------------------------+-----------------------+
+| ``RedirectionType``               | ``oo_redirection``                |redirection Form       |
++-----------------------------------+-----------------------------------+-----------------------+
+| ``RoleType``                      | ``oo_role``                       |roles Form             |
++-----------------------------------+-----------------------------------+-----------------------+
+| ``SiteAliasType``                 | ``oo_site_alias``                 |Add aliase Field       |
+|                                   |                                   |on SiteType            |
++-----------------------------------+-----------------------------------+-----------------------+
+| ``SiteType``                      | ``oo_site``                       |Site Form              |
++-----------------------------------+-----------------------------------+-----------------------+
+| ``StatusType``                    | ``oo_status``                     |status Form            |
++-----------------------------------+-----------------------------------+-----------------------+
+| ``TemplateType``                  | ``oo_template``                   |template Form          |
++-----------------------------------+-----------------------------------+-----------------------+
+| ``ThemeType``                     | ``oo_theme``                      |theme Form             |
++-----------------------------------+-----------------------------------+-----------------------+
+| ``TinymceType``                   | ``oo_tinymce``                    |                       |
++-----------------------------------+-----------------------------------+-----------------------+
+| ``TranslatedValueCollectionType`` | ``oo_translated_value_collection``|                       |
```

Comment:
This field is used to get a translatable text. It presents several tabs, one per language allowed in FO and allows the contributor to translate a text line in those languages.